lib/denylist/http.go

```diff
@@ -90,21 +90,23 @@ func getDenylistEntry(response http.ResponseWriter, request *http.Request, denyl
 func createDenylistEntry(response http.ResponseWriter, request *http.Request, denylist *sync.Map, syncer *Syncer) {
 	id := request.URL.Path
 	if strings.Contains(id, "/") {
+		log.Log.Warnw("Denylist PUT: entry includes '/'", "id", id)
 		http.Error(response, http.StatusText(http.StatusNotFound), http.StatusNotFound)
 		return
 	}
 	_, exists := denylist.Load(id)
 	if exists {
+		log.Log.Infow("PUT: Create called for denylist entry that already exists", "id", id)
```

Comment:
Just to keep formatting consistent.
```suggestion
		log.Log.Infow("Denylist PUT: Create called for denylist entry that already exists", "id", id)
```